.github/workflows/tests.yml

```diff
@@ -24,10 +24,13 @@ jobs:
           python-version: "3.11"
           architecture: x64
           cache: pip
-          cache-dependency-path: "**/*requirements*.txt"
+          cache-dependency-path: "**/pyproject.toml"
+      - name: Install Poetry
+        run: |
+          pip install poetry
       - name: Install dependencies
         run: |
-          pip install -r code/requirements.txt -r code/dev-requirements.txt -r code/backend/requirements.txt
+          pip install .
```

Comment:
I think we also need to install the `dev-dependencies`. I _think_ this only installs the `dependencies`